microprofile.cpp

```diff
@@ -8540,6 +8540,18 @@ void MicroProfileGpuInitVulkan(VkDevice* pDevices, VkPhysicalDevice* pPhysicalDe
 
 void MicroProfileGpuShutdown()
 {
+	VkAllocationCallbacks* noAlloc = nullptr;
```

Comment:
Implementing cleanup. without this my particular laptop will crash on shutdown

Reply:
looks good